WcaOnRails/app/views/competitions/_index_form.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
```

Comment:
Can we get the buttons to be side by side on mobile? This feels like a big waste of space currently.

By "anchor", do you mean the floatThead?